app/styles/components/shared/c-modal.scss

```diff
@@ -16,16 +16,14 @@
     background-color: $color-2;
     border: 1px solid rgba($color-3, .3);
     color: $color-4;
-    height: calc(100% - 60px);
-    max-width: 860px;
+    height: initial;
```

Comment:
We don't need this property.

But, we definitely need a `max-height` with an `overflow-y: auto;` set. Also, don't forget the `-webkit-overflow-scrolling: touch;`.